### ISSUE TITLE
Use `src` layout

### ISSUE THREAD

**Issue by Stranger6667** [django-money/django-money#578]
It will cause some conflicts with existing PRs but will bring a lot of benefits. The most important (from my pov) is more reliable tests that will run against the actually installed package, not the local tree

**Comment by benjaoming** [comment]
Agreed, it makes tests more reliable! I guess we've known this for a long time :)

I think for the PRs involved, they won't be seriously disrupted by it, git is pretty good at tracking file re-locations.
